cabal-testsuite/PackageTests/Init/init.test.hs

```diff
@@ -3,6 +3,11 @@ import Test.Cabal.Prelude
 main = cabalTest $
   withSourceCopyDir "app" $ do
     cwd <- fmap testSourceCopyDir getTestEnv
-    cabal "init" ["-n", "--exe", "--application-dir=app", "--main-is=Main.hs", "--", cwd]
+
+    buildOut <- withDirectory cwd $ do
```

Comment:
I'm not entirely sure on how `withSourceCopyDir` behaves, so the use of `withDirectory` might be reduntant.

Reply:
Me neither. As long as it works. :)